RELEASE-NOTES.md

```diff
@@ -1,5 +1,42 @@
 # Prefect Release Notes
 
+## Release 2.10.10
+
+### The Need for (CLI & import) Speed
+
+We wanted the CLI to be as fast as the rest of Prefect. Through a series of enhancements, we've sped up CLI performance by as much as 4x on some systems! See the following pull requests for more information on how we sped up the CLI prefect import:
+- Delay `apprise` imports — https://github.com/PrefectHQ/prefect/pull/9557
+- Defer import of `dateparser` — https://github.com/PrefectHQ/prefect/pull/9582
+- Defer loading of Prefect integrations until necessary — https://github.com/PrefectHQ/prefect/pull/9571
+- Add `Block.get_block_class_from_key` and replace external uses of `lookup_type` — https://github.com/PrefectHQ/prefect/pull/9621
+
+### Enhancements
+- Add SIGTERM handling for workers — https://github.com/PrefectHQ/prefect/pull/9530
+- Add ability to view table artifacts with NaN values in the Prefect UI — https://github.com/PrefectHQ/prefect/pull/9585
+- Update `prefect version` command to avoid creating the database if it does not exist — https://github.com/PrefectHQ/prefect/pull/9586
+- Use "eager" upgrade strategy when upgrading requirements in Docker images — https://github.com/PrefectHQ/prefect/pull/9640
+- Allow client retries when server SQLite database is busy — https://github.com/PrefectHQ/prefect/pull/9632
+- Add lock for update to flow run state transitions — https://github.com/PrefectHQ/prefect/pull/9590
+
+### Fixes
+- Fix bug where `SIGTERM` was not properly captured as a flow run crash for flow runs created by a deployment — https://github.com/PrefectHQ/prefect/pull/9543
+- Fix deadlock when logging is overriden from an asynchronous context — https://github.com/PrefectHQ/prefect/pull/9602
+- Fix orchestration race conditions by adding lock for update to flow run state transitions — https://github.com/PrefectHQ/prefect/pull/9590
```

Comment:
There are a few PRs related to orchestration race conditions 🤔 Wonder if that's worth mentioning in one place

Reply:
ah I think I duplicated one! removed!